docs/hu/docs/index.md

```diff
@@ -0,0 +1,463 @@
+
+## Vélemények
+
+"_[...] Mostanában nagyon sokat használom a **FastAPI**-t. [...] Tulajdonképpen azt tervezem, hogy a  csapatom összes **ML szolgáltatásához a Microsoftnál** ezt használjuk. Ezek közül néhányat integrálunk is magába a **Windows** termékbe és néhány **Office** termékbe is._"
+
+<div style="text-align: right; margin-right: 10%;">Kabir Khan - <strong>Microsoft</strong> <a href="https://github.com/tiangolo/fastapi/pull/26" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_Átvettük a **FastAPI** könyvtárat, hogy olyan **REST** kiszolgálót hozzunk létre, amelyet az **előrejelzések** lekérdezésére lehet használni. [a Ludwig számára]_"
+
+<div style="text-align: right; margin-right: 10%;">Piero Molino, Yaroslav Dudin, and Sai Sumanth Miryala - <strong>Uber</strong> <a href="https://eng.uber.com/ludwig-v0-2/" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_A **Netflix** örömmel jelenti be **válságkezelési** orchestration keretrendszerének nyílt forrású változatát: **Dispatch**! [**FastAPI**-val készült]_"
+
+<div style="text-align: right; margin-right: 10%;">Kevin Glisson, Marc Vilanova, Forest Monsen - <strong>Netflix</strong> <a href="https://netflixtechblog.com/introducing-dispatch-da4b8a2a8072" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_Nagyon lelkesít a **FastAPI**. Annyira szórakoztató!_"
+
+<div style="text-align: right; margin-right: 10%;">Brian Okken - <strong><a href="https://pythonbytes.fm/episodes/show/123/time-to-right-the-py-wrongs?time_in_sec=855" target="_blank">Python Bytes</a> podcast host</strong> <a href="https://twitter.com/brianokken/status/1112220079972728832" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_Őszintén szólva, amit felépítettél, az nagyon szilárdnak és kifinomultnak tűnik. Sok szempontból olyan, mint amilyennek a **Hug**-ot szeretném látni - nagyon inspiráló, hogy valaki ilyet készít._"
+
+<div style="text-align: right; margin-right: 10%;">Timothy Crosley - <strong><a href="https://www.hug.rest/" target="_blank">Hug</a> creator</strong> <a href="https://news.ycombinator.com/item?id=19455465" target="_blank"><small>(ref)</small></a></div>
+
+---
+
+"_Ha szeretnél megtanulni egy **modern keretrendszert** REST API-k létrehozásához, nézd meg a **FastAPI**-t [...] Gyors, egyszerűen használható és könnyen tanulható [...]_"
+
+"_Átálltunk a **FastAPI**-ra az **API**-jainknál [...] Szerintem kedvelni fogod [...]_"
+
+<div style="text-align: right; margin-right: 10%;">Ines Montani - Matthew Honnibal - <strong><a href="https://explosion.ai" target="_blank">Explosion AI</a> founders - <a href="https://spacy.io" target="_blank">spaCy</a> creators</strong> <a href="https://twitter.com/_inesmontani/status/1144173225322143744" target="_blank"><small>(ref)</small></a> - <a href="https://twitter.com/honnibal/status/1144031421859655680" target="_blank"><small>(ref)</small></a></div>
+
+---
+
```

Comment:
"_Ha bárki egy production Python API-t tervez építeni, én nagyon ajánlom a **FastAPI**-t.  **Gyönyörűen kialakított**, **egyszerűen használható** és **könnyen skálázható**, **kulcsfontosságú**  lett az API-t előtérbe helyező fejlesztési stratégiánknak, valamint számos automatizálás és szolgáltatás mögött is ő áll, mint például a Virtual TAC Engineer-ünk._"

<div style="text-align: right; margin-right: 10%;">Deon Pillsbury - <strong>Cisco</strong> <a href="https://www.linkedin.com/posts/deonpillsbury_cisco-cx-python-activity-6963242628536487936-trAp/" target="_blank"><small>(ref)</small></a></div>

---